content/blog/2021-02-17-cml-runner-prerelease.md

```diff
@@ -0,0 +1,165 @@
+---
+title: 'Pre-Release Notes: Automatically Train Models in the Cloud with CML'
+date: 2021-02-17
+description: |
+  New features are here to make launching cloud 
+  compute for continuous integration workflows 
+  shorter, sweeter and easier than ever. Plus, 
+  a new GitHub Action to setup CML means more 
+  ways to use CML without our Docker container.
+descriptionLong: |
+  New features are here to make launching cloud 
+  compute for continuous integration workflows 
+  shorter, sweeter and easier than ever. Plus, 
+  a new GitHub Action to setup CML means more 
+  ways to use CML without our Docker container.
+picture: 2021-02-17/gh_actions_art.png
+author: elle_obrien
+commentsUrl: https://discuss.dvc.org/t/cml-self-hosted-runners-on-demand-with-gpus/462
+tags:
+  - CML
+  - Continuous Machine Learning
+  - GitHub Actions
+  - GitLab
+  - Cloud
+---
+
+Today, we're pre-releasing some new features in Continuous Machine Learning, or
+[CML](https://cml.dev)- our open source project to adapt popular continuous
+integration (CI) systems like GitHub Actions and GitLab CI for data science.
+
+Here's what you'll get:
+
+## Brand new method to provision cloud compute for your CI workflows
```

Comment:
It might be not easy to understand for readers who didn't use the CML before. Does it make sense to give a short intro and show a PR report image?  `CML become a popular tool for auto-generating ML model reports right in GitHub Pull Request (see in the image below) and a way of allocating model training resources in clouds`.

Reply:
the owl report might look nice here

Reply:
to me it's too confusing to add the owl PR picture, since we're going into a workflow that produces a completely different picture. I will add some more introduction to what CML is in the intro though.